__tests__/app.test.js

```diff
@@ -63,7 +63,7 @@ describe('app.all handles non-existent paths', () => {
 });
 
 describe("GET /api/articles", () => {
-  test('200: returns an array of article objects', () => {
+  test('200: returns an array of article objects with added comment count', () => {
```

Comment:
as we are asserting inside a forEach, we first need to check if the length is what we are expecting - if the length is 0 this test will pass by default as none of the expect statements will run!